contrib/win32/win32compat/pwd.c

```diff
@@ -262,6 +262,32 @@ w32_getpwuid(uid_t uid)
 	return ret;
 }
 
+PSID
+getusid(void)
+{
+	LPWSTR user_sid = NULL;
+	HANDLE token = 0;
+	DWORD info_len = 0;
+	TOKEN_USER* info = NULL;
+	PSID ret = NULL;
+
+	errno = 0;
+
+	if (OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &token) == FALSE ||
+		GetTokenInformation(token, TokenUser, NULL, 0, &info_len) == TRUE ||
```

Comment:
What is your concern here? FALSE is error condition to per msdn, If the function succeeds, the return value is nonzero.
If the function fails, the return value is zero. To get extended error information, call GetLastError.
I will change to == 0 which may be clearer